docs/antrea-ipam.md

```diff
@@ -193,18 +194,26 @@ restore its configurations at exit. Node may lose network connection when `antre
 or OVS daemons are stopped unexpectedly, which can be recovered by rebooting the Node.
 `AntreaIPAM` Pods' traffic will not be routed by local Node's network stack.
 
-All traffic to a local Pod will be sent to the Pod's OVS port directly, after the
-destination MAC is rewritten to the Pod's MAC address. This includes `AntreaIPAM` Pods
-and regular `Subnet per Node` IPAM Pods, even they are not in the same subnets.
-Inter-Node traffic will be sent to the Node network from the source Node, and forwarded
-to the destination Node by the Node network.
+Traffic from `AntreaIPAM` Pods without VLAN, regular `Subnet per Node` IPAM Pods, and K8s
+Nodes are recognized as VLAN 0 (untagged).
+
+All traffic to a local Pod in same VLAN will be sent to the Pod's OVS port directly,
+after the destination MAC is rewritten to the Pod's MAC address. This includes
+`AntreaIPAM` Pods and regular `Subnet per Node` IPAM Pods, even they are not in the same
+subnets.
+All traffic to a Pod in different VLAN will be sent to the gateway of "underlay" network.
+Inter-Node traffic will be sent to the "underlay" network from the source Node, and
+forwarded to the destination Node by the "underlay" network.
```

Comment:
I think quotes are not needed with `underlay` as those are not used in other places (like below)

Reply:
Thanks, removed quotes.